docs-conceptual/azps-2.0.0/new-azureps-module-az.md

```diff
@@ -22,12 +22,25 @@ Az is a new module, so the version has been reset to 1.0.0.
 
 ## Upgrade to Az
 
-It's recommended that all users upgrade to the new Az module. To do so:
+To keep up with the latest Azure features in PowerShell, you should migrate to the Az module as soon as
+possible. If you're not ready to install the Az module as a replacement for AzureRM, you have a couple of
+options available to experiment with Az:
 
-* __RECOMMENDED__: [Uninstall the Azure PowerShell AzureRM module](/powershell/azure/uninstall-az-ps#uninstall-the-azurerm-module)
-* [Install the Azure PowerShell Az module](/powershell/azure/install-az-ps)
-* Enable compatibility mode to add aliases for AzureRM cmdlets with `Enable-AzureRMAlias`
-  while you become familiar with the new command set. __Only__ enable aliases if you do not have AzureRM installed.
+* Use a `PowerShell` environment with [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/overview).
+  Azure Cloud Shell is an environment which comes with the Az module installed, `Enable-AzureRM` compatibility aliases
+  enabled, and the [Azure CLI](/cli/azure).
```

Comment:
The reference to Azure CLI may be confusing in this page about the Az module for PowerShell.

Reply:
I'm tempted to leave it in so that people are aware of the CLI as an alternative, but we don't have any actual information on if users know about only one or the other, or both, and why they choose to use one.